### DISCUSSION TITLE
[M-Z] (12 Settembre 2018) Esercizio 5

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#497] [Esami & Esoneri M-Z]
### Traccia

<img width="964" alt="image" src="https://github.com/user-attachments/assets/09cdd5ef-28a8-48b6-97c4-03c1c7be17eb">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20180912.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20180912.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/add83587-ee3b-47a4-8e24-4847b82cb4a9)
